doc/source/train/api.rst

```diff
@@ -132,16 +173,32 @@ Mosaic
     :show-inheritance:
 
 
-Reinforcement Learning (RLlib)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Reinforcement Learning with RLlib
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+``RLTrainer``
```

Comment:
The RLTrainer links are no longer working 
<img width="762" alt="image" src="https://user-images.githubusercontent.com/8068268/212180869-d612948b-1677-4128-b107-5eb9a335db4d.png">

Reply:
good catch, thanks